3.1/bionic/Dockerfile

```diff
@@ -6,6 +6,7 @@
 
 ARG BASE_IMAGE=rstudio/r-base
 FROM ${BASE_IMAGE}:bionic
+LABEL maintainer="RStudio Docker <docker@rstudio.com>"
```

Comment:
I believe that labels are inherited. Why not just put the maintainer label on the the base image instead of each R version?